index.html

```diff
@@ -4,10 +4,176 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="mobile-version-style.css">
+    <title>My Portfolio Mobile Version</title>
+    <link rel="shortcut icon" href="Images/favicon.ico"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;700&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+    <main id="card-container">
+        <div class="card-works">
+            <div class="card-work-insider">
+                 <img class="image-snapshot" src="Images/Snapshoot Portfolio.png" alt="Work 1 Tonic Portfolio">
+                 <h2 class="project-title">Tonic</h2>
+                 <ul class="frame-2">
+                    <li>CANOPY</li>
+                    <li><img src="Images/Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="Images/Counter.png"></li>
+                    <li>2015</li>
+                 </ul>
+                 <p class="primary-project-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <ul class="tags-container">
+                    <li class="tags">html</li>
+                    <li class="tags">CSS</li>
+                    <li class="tags">javaScript</li>
+                 </ul>
+                 <div class="action-button">
+                    <button class="see-project-button">See Project</button>
+                 </div>
+            </div>
+        </div>
```

Comment:
- [ ] Good job implementing the works section 👍 , but it could be better if you add some `margin` to stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=t6rOGNfHIv0gVnpn-0) as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃 



> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | ![cart01](https://user-images.githubusercontent.com/57408419/221091556-49a2a195-1606-4986-bb1d-e4455e1c5158.jpg)| |![cart](https://user-images.githubusercontent.com/57408419/221091620-a348abac-a5b7-44ad-af01-8119a80699c7.jpg) | |